pegasus/src/homepage.rb

```diff
@@ -129,6 +129,14 @@ def self.get_actions
           url: "/hourofcode/overview"
         }
       ]
+    elsif hoc_mode == "soon-hoc" && show_single_hero(request) == "oceans2019"
+      [
+        {
+          text: "homepage_action_text_try_it",
+          type: "cta_button_hollow_white",
+          url: "/hourofcode/overview"
```

Comment:
Are you sure this shouldn't go straight to code.org/oceans?

Reply:
We will continue showing the existing `"soon-hoc"` buttons.